bld/ExtensionsGenerator/MoreLinq.ExtensionsGenerator.csproj

```diff
@@ -6,7 +6,13 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="docopt.net" Version="0.8.1" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.9.2" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.11.0" />
     <PackageReference Include="System.Collections.Immutable" Version="8.0.0" />
   </ItemGroup>
+  <ItemGroup>
+    <EditorConfigFiles Remove="A:\MoreLINQ\main\bld\ExtensionsGenerator\.editorconfig" />
```

Comment:
Recommend not to hard-code paths in .csproj, and use relative paths only.

Reply:
Yeah, I don't know why VS decided to do that. Bizarre. Removed.